Makefile

```diff
@@ -1,8 +1,8 @@
 CFLAGS = -Wall -Werror -fpic -std=gnu99
 
 ifeq ($(OS),Windows_NT)
-    MKD = cmd /C mkdir
```

Comment:
@Conni2461 this seems to cause issues on windows, no idea why it was changed, but the old version worked just fine i think. With that commit make fails to build

Reply:
Like i said here: https://github.com/nvim-telescope/telescope-fzf-native.nvim/pull/87#issuecomment-1345948769 i am aware of this issue.

But i can't test anything because i am not on a windows machine. This PR fixed fix https://github.com/nvim-telescope/telescope-fzf-native.nvim/issues/82

And my last question was, does #89 fix, #82 and the new issues #88, #90 ?

I needed someone to confirm this, but tbh no one did so i'am just gonna merge